File: app/src/main/java/com/kee0kai/thekey/utils/views/EmptyTextWatcher.java
package com.kee0kai.thekey.utils.views;

import android.text.Editable;
import android.text.TextWatcher;

public class EmptyTextWatcher implements TextWatcher {

    public boolean ignoreChanges = false;

    @Override
    public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {

    }

    @Override
    public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {

    }

    @Override
    public void afterTextChanged(Editable editable) {

    }
}

File: core/src/main/kotlin/com/github/klee0kai/thekey/core/utils/views/EmptyTextWatcher.kt
package com.github.klee0kai.thekey.core.utils.views

import android.text.Editable
import android.text.TextWatcher

open class EmptyTextWatcher : TextWatcher {
    var ignoreChanges = false

    override fun beforeTextChanged(s: CharSequence?, start: Int, count: Int, after: Int) {

    }

    override fun onTextChanged(s: CharSequence?, start: Int, before: Int, count: Int) {

    }

    override fun afterTextChanged(s: Editable?) {

    }
}